src/screening/vl/VLForm.jsx

```diff
@@ -110,9 +110,23 @@ class VLForm extends React.PureComponent {
   }
 
   render() {
-    const { vlResultStatus, vlLowerthanDetectableLimits, reasonForTesting, labLocation, bled, reasonForNoResult, vlDetectableLowerLimit, vlNumeric } = this.props;
+    const {
+      vlResultStatus,
+      vlLowerthanDetectableLimits,
+      reasonForTesting,
+      labLocation,
+      bled,
+      reasonForNoResult,
+      vlDetectableLowerLimit,
+      vlNumeric,
+      vlLessThanLimit,
+    } = this.props;
     const { isAddVLResults, isAddVLResultsClicked, isAddVLResultsActive } = this.state;
 
+    const showVlResultValidations =  !((vlResultStatus.value === CONCEPTS.ViralLoadResultCompleted.uuid || vlResultStatus.value === CONCEPTS.ViralLoadResultUnableToProcess.uuid)  &&
```

Comment:
@rhenshaw56 wouldn't this better be called just "vlResultValidations", or am I missing something?